js/main.js

```diff
@@ -0,0 +1,39 @@
+let string = 'some test string';
+
+alert(string[0]); //получаем первую букву
+alert(string.length); //определяем длину
+alert(string.length-1); //последний символ строки
+alert(string[15]); //получаем последнюю буквы
+alert(string[0].toUpperCase()); //делаем первую букву большой
+alert(string[15].toUpperCase()); //делаем последнюю букву большой
+alert(string.indexOf("string")); //находим положение слова в строке
+alert(string.indexOf("string")-1); //положение второго пробела
+alert(string.substr(5,4));// получаем строку с 5 символа 4 буквы
+alert(string.slice(5,9));// получаем строку с 5 по 9 символы
+alert(string.slice(0, -6) );// новая строка с удалением 6 символов из предыдущей строки
+var a = '20';
+	b = '16';
```

Comment:
`a` and `b` should have been number type. Anf you should have done something like this `var a = 20; var b= 16; alert(`${a}${b}`)`  - The most correct way would be using template string here